src/main/java/de/tum/in/www1/artemis/domain/Submission.java

```diff
@@ -76,6 +76,10 @@ public abstract class Submission extends DomainObject implements Comparable<Subm
     @Column(name = "submission_date")
     private ZonedDateTime submissionDate;
 
+    @Column(name = "plagiarism_detected")
+    @JsonView(QuizView.Before.class)
+    private Boolean plagiarismDetected;
```

Comment:
```suggestion
    private boolean plagiarismDetected;
```
Can we use the primitive here or do you need null values as well?

Reply:
There is a default value in the db so nullability is not needed. Changed to `boolean`